src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ReceivedMessagesHealthController.java

```diff
@@ -113,7 +119,18 @@ private boolean isDateWithinWorkingHours(LocalDateTime localDateTime) {
             && (localDateTime.equals(workingHoursEndTime) || localDateTime.isBefore(workingHoursEndTime));
     }
 
-    private boolean isNotEnabledForEnvironment(String env) {
+    public boolean isNotEnabledForEnvironment(String env) {
+        if (ENV_AAT.equals(env)) {
+
+            URI currentUri = ServletUriComponentsBuilder.fromCurrentRequestUri().build().toUri();
+            String currentUriString = "";
+            if (null != currentUri) {
+                currentUriString = currentUri.toString();
+            }
+            if (currentUriString.contains(STAGING_TEXT)) {
+                return true;
+            }
```

Comment:
I don't think you need the currentUriString var since you have currentUri.toString?

Reply:
Please can you out put the url string so we can see what it is in logs as mentioned above please?

Reply:
I added the logs Ravi.